src/science/libndtp/ndtp.cpp

```diff
@@ -177,7 +177,7 @@ ByteArray NDTPPayloadSpiketrain::pack() const {
   result.push_back(bin_size_ms);
 
   // pack clamped spike counts
-  auto [bytes, final_bit_offset] = to_bytes(clamped_counts, BIT_WIDTH_BINNED_SPIKES, {}, 0);
+  auto [bytes, final_bit_offset, _] = to_bytes(clamped_counts, BIT_WIDTH_BINNED_SPIKES, {}, 0);
```

Comment:
imo we should update the signature of these pack() functions to return a status bool and populate a bytearray by ptr, or something, to propagate the status up -- otherwise this still fails silently. could add in a different pr